lib/Net/Async/Redis.pm

```diff
@@ -1161,11 +1161,12 @@ sub execute_command {
         $self->{connection_in_progress}
         ? Future->done
         # Are we the owner of a current MULTI transaction?
-        : $self->{_is_multi}
```

Comment:
Here should be:
if it is multi, then should test `self->connected`, and then wait `$self->{pending_multi}`
if it is not multi, then just test `$self->connected`